### PR TITLE
TAJO-1963: Add more configuration descriptions to document

### PR DESCRIPTION
**Pull request by jihoonson**
I also fixed a wrong configuration name.

### CODE REVIEW

**Comment by jihoonson** [comment]
You can see the updated document here.
http://people.apache.org/~jihoonson/tajo-docs/

File: tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst [hunk 1]
```diff
@@ -2,23 +2,455 @@
...
+Join Query Settings
+=========================
+
+""""""""""""""""""""""""""""""""""""""
+`tajo.dist-query.join.auto-broadcast`
+""""""""""""""""""""""""""""""""""""""
+
+A flag to enable or disable the use of broadcast join.
+
+  * Property value: Boolean
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst]: IMO, 'property value type' looks clearer.

File: tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst [hunk 2]
```diff
@@ -2,23 +2,455 @@
...
+
+  * Property value: Integer
+  * Unit: MB
+  * Default value: 64
+  * Example
+
+.. code-block:: xml
+
+  <property>
+    <name>tajo.dist-query.groupby.partition-volume-mb</name>
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst]: The title is written with 'task', but example is done with 'partition'

File: tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst [hunk 3]
```diff
@@ -2,23 +2,455 @@
...
+.. warning::
+  This value is the size of the input stored on file systems. So, when the input data is loaded into JVM heap,
+  its actual size is usually much larger than the configured value, which means that too large threshold can cause unexpected OutOfMemory errors.
+  This value should be tuned carefully.
+
+""""""""""""""""""""""""""""""""""""""""""
+`tajo.executor.aggregate.hash-table.size`
+""""""""""""""""""""""""""""""""""""""""""
+
+The initial size of list for in-memory sort.
```
Comment (eminency) [inline tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst]: Description explains for list size, but property name looks that it means hash table size.

**Comment by eminency** [comment]
I leave some comments.

File: tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst (hunk 1 above)
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst]: Thank you for the good comment.

File: tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst (hunk 2 above)
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst]: My mistake. Thanks.

File: tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst (hunk 3 above)
Comment (jihoonson) [inline tajo-docs/src/main/sphinx/configuration/tajo-site-xml.rst]: My mistake. Thanks.

**Comment by jihoonson** [comment]
Thanks for your comment. I addressed your comments.

**Comment by eminency** [comment]
Thanks, it looks good. +1

**Comment by hyunsik** [comment]
Could check the unit test?
```
Failed tests: 
  TestTajoCli.testHelpSessionVars:410->assertOutputResult:103->assertOutputResult:107->assertOutputResult:125 expected:<...able size
\set SORT_[HASH_TABLE_SIZE [int value] - Sort hash table size
\set JOIN_HASH_TABLE_SIZE [int value] - Join hash table size
\set INDEX_ENABLED [true or false] - index scan enabled
\set INDEX_SELECTIVITY_THRESHOLD [real value] - the selectivity threshold for index scan
\set PARTITION_NO_RESULT_OVERWRITE_ENABLED [true or false] - If T]rue, a partitioned t...> but was:<...able size
\set SORT_[LIST_SIZE [int value] - List size for in-memory sort 
\set JOIN_HASH_TABLE_SIZE [int value] - Join hash table size
\set INDEX_ENABLED [true or false] - index scan enabled
\set INDEX_SELECTIVITY_THRESHOLD [real value] - the selectivity threshold for index scan
\set PARTITION_NO_RESULT_OVERWRITE_ENABLED [true or false] - If t]rue, a partitioned t...>
Tests run: 1593, Failures: 1, Errors: 0, Skipped: 0
```

**Comment by jihoonson** [comment]
@eminency and @hyunsik, thank you guys for your review!
I fixed the test failure.

**Comment by hyunsik** [comment]
+1
The patch looks good to me.

**Pull request closed without merging**